docs/how-tos/deploy-appgw-and-agic-in-azure-network.md

```diff
@@ -0,0 +1,58 @@
+## Overview
+
+AGIC(Application Gateway Ingress Controllor) is a pod running in AKS/Kubernetes cluster, to make AGIC work properly with AppGw(Azure Application Gateway), there are multiple ways to deploy them in Azure network:
```

Comment:
When setting up network connectivity, our focus is to get network access from AppGW to AKS cluster (Backend pods). Not between AGIC and AppGW.
Talking about where AGIC is running w.r.t AppGW will cause confusion.